modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/client/strategy/OperationMessageProcessorStrategy.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.runtime.client.strategy;
+
+import org.mule.api.annotation.NoImplement;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.extension.api.client.OperationParameters;
+import org.mule.runtime.module.extension.internal.runtime.operation.OperationMessageProcessor;
+
+/**
+ * Interface that provides help for executing extension operations.
+ *
+ * @since 4.1.6
+ */
+@NoImplement
+public interface OperationMessageProcessorStrategy {
```

Comment:
I'd rename OperationMessageProcessorStrategy to something much more specific to the EC, like ExtensionsClientProcessorsStrategy

Reply:
Agree